libvast/src/system/index.cpp

```diff
@@ -672,6 +686,7 @@ void index_state::send_report() {
   auto query_counters = get_query_counters(pending_queries);
   auto msg = report{
     .data = {
+      {"index.partitions-created", counters.active_partitions_created},
```

Comment:
What value does this metric add for the operator? It's just the number of active partitions created since the start of VAST, right?